src/LoadingPage/index.js

```diff
@@ -3,7 +3,7 @@ import { Container, StyledParagraph, IconContainer, Icon } from "./styled";
 export const LoadingPage = ({ title }) => (
   <Container>
     <StyledParagraph>
-      {title ? <>Search results for {title}</> : ""}
+      {title ? <>Search results for "{title}"</> : ""}
```

Comment:
If you only need to render content when title is present and don't need to handle the case when title is missing, you can simplify your code by using the && operator for conditional rendering😎. This method is cleaner:)
```js
<StyledParagraph>
  {title && <span>Search results for "{title}"</span>}
</StyledParagraph>
```

Reply:
Great solution, thank you for the tip.